### PR TITLE
Namco system 246/256 support

### PR DESCRIPTION
**Pull request by israpps**
~~Like many applications compiled on latest SDK. Something is preventing the code execution from reaching `main()` when using 2x6 bios on pcsx2.~~

The changes so far  **should** suffice to at least get the BG image on screen on real hardware (?)

### CODE REVIEW

**Comment by AKuHAK** [comment]
@israpps I switched to the bdm, so conflicts apeeared

**Comment by israpps** [comment]
> @israpps I switched to the bdm, so conflicts apeeared

Fixed!

**Comment by AKuHAK** [comment]
fail

**Comment by israpps** [comment]
<details>
  <summary> <b> Emulator test</b> </summary>
<p>

_abe64c7b_20240417113011
_abe64c7b_20240417113013
_abe64c7b_20240417113015
_abe64c7b_20240417113017
_abe64c7b_20240417113020
_abe64c7b_20240417113022
_abe64c7b_20240417113024
_abe64c7b_20240417113027


</p>
</details>

**Comment by israpps** [comment]
enable CI pls

File: Makefile [hunk 1]
```diff
@@ -1,38 +1,58 @@
...
+  EE_CFLAGS += -DDSNET_HOST_SUPPORT
+  DEBUG = 1
 endif
 
 ifeq ($(DEBUG),1)
-	IOP_CFLAGS += -DDEBUG
-	EE_CFLAGS += -DDEBUG
+  IOP_CFLAGS += -DDEBUG
+  EE_CFLAGS += -DDEBUG
+
```
Comment (AKuHAK) [inline Makefile:43]: Missing endif

**Comment by AKuHAK** [comment]
> enable CI pls

Isn't it easier to enable CI on your fork?

**Comment by AKuHAK** [comment]
sorry, conflicts

**Comment by israpps** [comment]
XD
image

**Pull request merged**